### ISSUE TITLE
[🐛] Local Notification

### ISSUE THREAD

**Issue by SSinfo302** [invertase/react-native-firebase#5128]
**Local notification was in v5.**
const localNotification = new firebase.notifications.Notification({
                            sound: ‘default’,
                            show_in_foreground: true,
                        })
                            .setNotificationId(notification.notificationId)
                            .setTitle(notification.title)
                            .setSubtitle(notification.subtitle)
                            .setBody(notification.body)
                            .setData(notification.data)
                            .android.setChannelId(‘code-chanel’) // e.g. the id you chose above
                            .android.setColor(‘#FFFFFF’) // you can set a color here
                            .android.setPriority(firebase.notifications.Android.Priority.High)
                            .android.setBigText(notification.body)
                            .android.setSmallIcon(‘app_icon’)
                            .android.setAutoCancel(true);
                        firebase.notifications()
                            .displayNotification(localNotification)
                            .catch(err => console.log(err));

**how can implement in v6?**

**Comment by andersonaddo** [comment]
All that is explained over here:
https://rnfirebase.io/messaging/usage

TLDR:
No longer supported, use third party packages now
